uber/models.py

```diff
@@ -986,7 +986,7 @@ class Attendee(MagModel, TakesPaymentMixin):
     amount_paid      = Column(Integer, default=0, admin_only=True)
     amount_extra     = Column(Choice(c.DONATION_TIER_OPTS, allow_unspecified=True), default=0)
     amount_refunded  = Column(Integer, default=0, admin_only=True)
-    payment_method   = Column(Choice(c.PAYMENT_METHOD_OPTS), nullable=True, admin_only=True)
+    payment_method   = Column(Choice(c.PAYMENT_METHOD_OPTS), nullable=True)
```

Comment:
I forgot that this is something we explicitly ask for on the at-the-door reg form and therefore it is NOT admin only.  (That's only checked in the Python code and isn't part of the db, so this change doesn't require a migration or anything.)